book.rb

```diff
@@ -0,0 +1,15 @@
+require_relative 'rental'
+
+class Book
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
```

Comment:
- [ ] Before merging, please make sure that you add getter and setter for `@rentals` via attr_accessor. It will help you to fetch @rentals's value and to modify it.